task-definition.worker.json

```diff
@@ -0,0 +1,205 @@
+                },
+                {
+                    "name": "SOCIAL_AUTH_GITHUB_SECRET",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:SOCIAL_AUTH_GITHUB_SECRET::"
+                },
+                {
+                    "name": "SOCIAL_AUTH_GITLAB_KEY",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:SOCIAL_AUTH_GITLAB_KEY::"
+                },
+                {
+                    "name": "SOCIAL_AUTH_GITLAB_SECRET",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:SOCIAL_AUTH_GITLAB_SECRET::"
+                },
+                {
+                    "name": "SOCIAL_AUTH_GOOGLE_OAUTH2_KEY",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:SOCIAL_AUTH_GOOGLE_OAUTH2_KEY::"
+                },
+                {
+                    "name": "SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET::"
+                },
+                {
+                    "name": "STATSD_HOST",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:STATSD_HOST::"
+                },
+                {
+                    "name": "STRIPE_API_KEY",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:STRIPE_API_KEY::"
+                },
+                {
+                    "name": "STRIPE_DEFAULT_PRICE_ID",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:STRIPE_DEFAULT_PRICE_ID::"
+                },
+                {
+                    "name": "STRIPE_GROWTH_PRICE_ID",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:STRIPE_GROWTH_PRICE_ID::"
+                },
+                {
+                    "name": "STRIPE_PUBLISHABLE_KEY",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:STRIPE_PUBLISHABLE_KEY::"
+                },
+                {
+                    "name": "STRIPE_WEBHOOK_SECRET",
+                    "valueFrom": "arn:aws:secretsmanager:us-east-1:795637471508:secret:Posthog_Production_Heroku-FQ2itU:STRIPE_WEBHOOK_SECRET::"
+                }
+            ],
+            "entryPoint": ["sh", "-c"],
+            "command": ["./bin/docker-worker"]
+        }
+    ],
+    "requiresCompatibilities": ["FARGATE"],
+    "cpu": "512",
+    "memory": "1024"
```

Comment:
don't know much about this but should these values be the same across all definitions

Reply:
To start out with that's the idea. That will probably change later as we watch the metrics.